src/App/Expression.lex.c

```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Confirming: Even though Flex is BSD-licensed, it makes no restrictions on the licensing of the generated scanner, so we are free to choose LGPL-2.1-or-later.